### DISCUSSION TITLE
Flexget rejects downloadging the release from second group specified in series plugin

### DISCUSSION THREAD

**Discussion by bodziek666** [Flexget/Flexget#3328] [Q&A] [answered]
Hi.

I don't understand why, but flexget rejects downloading torrent from Group B when torrent from Group A for a specific series has been already downloaded:
```
2022-01-26 10:16:04 VERBOSE  task          airing-anime REJECTED: `My Dress-Up Darling - TV Series [2022] :: Web | MKV | h265 10-bit | 1080p | AC3 2.0 | Softsubs (GJM) | Episode 3 | Freeleech` by series plugin because entity has already been downloaded
```

The configuration file looks like this:
```
tasks:
  airing-anime:
    rss: '{? airing_anime_rss.url ?}'
    series:
      settings:
        tv:
          set:
            path: /mnt/das0_data/downloads/airing/
      tv:
        - 'Demon Slayer: Kimetsu no Yaiba - Entertainment District Arc':
            quality: 1080p
            from_group: SubsPlease
        - 'Slow Loop':
            quality: 1080p
            from_group: SubsPlease
        - "My Dress-Up Darling":
            quality: 1080p
            from_group:
              - SubsPlease
              - GJM
```
Is there anything I can do to download them both?

**Comment by bodziek666** [comment, marked as answer]
I think I've found a solution: https://flexget.com/Plugins/series/upgrade

Basically:
```
        - "My Dress-Up Darling":
            quality: 1080p
            upgrade: yes
            from_group:
              - SubsPlease
              - GJM
```
worked out in my case.

**Comment by tubedogg** [comment]
FlexGet normally only accepts the first copy of an episode that it sees that matches the requirements in the config. `from_group` means that whatever is ultimately downloaded must come from one of those groups, not that it will download an episode twice to match each group.

`upgrade` will only work if the release from Group B is in some way of a [higher quality](https://flexget.com/Qualities) (in your example from your first post, depending how FlexGet is interpreting "web", HDTV may be considered higher quality, and DVD or BluRay will definitely be considered higher quality).

If you want it to accept a different version of the episode later, you would need to configure it to ignore the fact it's seen that episode before. I believe `if` should override the `series` plugin's rejection.

```
tasks:
  airing-anime:
    rss: '{? airing_anime_rss.url ?}'
    if:
      - "'My Dress-Up Darling' in series_name and release_group and release_group in ['SubsPlease', 'GJM']": accept
    series:
      settings:
        tv:
          set:
            path: /mnt/das0_data/downloads/airing/
      tv:
        - 'Demon Slayer: Kimetsu no Yaiba - Entertainment District Arc':
            quality: 1080p
            from_group: SubsPlease
        - 'Slow Loop':
            quality: 1080p
            from_group: SubsPlease
        - "My Dress-Up Darling":
            quality: 1080p
```
